pkg/controller/injector/k8s_binding_injector.go

```diff
@@ -27,7 +27,7 @@ import (
 )
 
 type k8sBindingInjector struct {
-	client *kubernetes.Clientset
+	client kubernetes.Interface
```

Comment:
I'm not sure there really should be any? Interface is the interface that Clientset implements. While this would cause trouble if we wanted to use features of the Clientset that aren't part of Interface, the fact there were no compile errors leads me to think this isn't the case.

The only reason I changed it here was so I could use a fake Clientset for unit tests